config/initializers/assets.rb

```diff
@@ -14,3 +14,5 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+
+Rails.application.config.assets.precompile += %w(app.js charts.js forms.js maps.js tables.js app.css *.png *.gif *.jpg *.ico *.svg *.eot *.ttf *.woff *.woff2)
```

Comment:
Can't you just add the vendor directory here ?

Reply:
couldn't find on stacko (how to write this line with a path), but all those extensions are looked for in vendor/assets (environment path)